std_msgs/Header header
uint32 width
uint32 height
event_based_camera/Event[] events
